docs/content/project/contributing.md

```diff
@@ -0,0 +1,253 @@
+      <td>Organize or attend a Paimon Meetup, contribute to the Paimon blog, share your conference, meetup or blog
+          post on the <a href="https://github.com/apache/incubator-paimon#mailing-lists">dev@paimon.apache.org mailing list</a>.
+      </td>
+    </tr>
+    <tr>
+      <td colspan="2">
+        <span class="glyphicon glyphicon-question-sign" aria-hidden="true"></span> Any other question? Reach out to the
+                     <a href="https://github.com/apache/incubator-paimon#mailing-lists">dev@paimon.apache.org mailing list</a> to get help!
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Code Contribution Guide
+
+Apache Paimon is maintained, improved, and extended by code contributions of volunteers. We welcome contributions to Paimpn.
+
+Please feel free to ask questions at any time. Either send a mail to the Dev mailing list or comment on the issue you are working on.
+
+<style>
+.contribute-grid {
+  margin-bottom: 10px;
+  display: flex;
+  flex-direction: column;
+  margin-left: -2px;
+  margin-right: -2px;
+}
+
+.contribute-grid .column {
+  margin-top: 4px;
+  padding: 0 2px;
+}
+
+@media only screen and (min-width: 480px) {
+  .contribute-grid {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .contribute-grid .column {
+    flex: 0 0 50%;
+  }
+
+  .contribute-grid .column {
+    margin-top: 4px;
+  }
+}
+
+@media only screen and (min-width: 960px) {
+  .contribute-grid {
+    flex-wrap: nowrap;
+  }
+
+  .contribute-grid .column {
+    flex: 0 0 25%;
+  }
+}
+
+.contribute-grid .panel {
+  height: 100%;
+  margin: 0;
+}
+
+.contribute-grid .panel-body {
+  padding: 10px;
+}
+
+.contribute-grid h2 {
+  margin: 0 0 10px 0;
+  padding: 0;
+  display: flex;
+  align-items: flex-start;
+}
+
+.contribute-grid .number {
+  margin-right: 0.25em;
+  font-size: 1.5em;
+  line-height: 0.9;
+}
+</style>
+
+<div class="contribute-grid">
+  <div class="column">
+    <div class="panel panel-default">
+      <div class="panel-body">
+        <h2><span class="number">1</span><a href="#consensus">Discuss</a></h2>
+        <p>Create a Issue or mailing list discussion and reach consensus</p>
+        <p><b>To request an issue, please note that it is not just a "please assign it to me", you need to explain your understanding of the issue, and your design, and if possible, you need to provide your POC code.</b></p>
+      </div>
+    </div>
+  </div>
+  <div class="column">
+    <div class="panel panel-default">
+      <div class="panel-body">
+        <h2><span class="number">2</span><a href="#implement">Implement</a></h2>
+        <p>Create the Pull Request and the approach agreed upon in the issue.</p>
+        <p><b>1.Only create the PR if you are assigned to the issue. 2.Please associate an issue (if any), e.g. fix #123. 3.Please enable the actions of your own clone project, show a screenshot of the test passing.</b></p>
```

Comment:
Why we must need to `show a screenshot of the test passing`?

Reply:
For non-apache-committers, only testing run in their own github repo. It is good to show the test passing.

Reply:
OK it is ugly, reviewer can take a look to corresponding link to see test passing.